src/components/veocean/WithdrawOcean.svelte

```diff
@@ -29,6 +34,16 @@
       async () => {
         loading = false;
         await addUserOceanBalanceToBalances($connectedChainId);
+        const newVeOceansWithDelegations =
+          await getUserVotingPowerWithDelegations($userAddress);
```

Comment:
Although it's good that we're adding this, we may still want to disable voting power from the dashboard.

I.E. Users won't be able to delegate via the FE for the next little while.

Reply:
Yes, remember that, let's handle it into another PR since we have to remove it from multiple places.
For the completion of this PR let's keep it like this to verify that everything is updating correctly.

Reply:
100%